website/pages/docs/from-1.5/blocks/packer.mdx

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+page_title: packer - Blocks
+sidebar_title: <tt>packer</tt>
+description: |-
+  The "packer" configuration section is used to configure some behaviors
+  of Packer itself.
+---
+
+# Packer Settings
+
+`@include 'from-1.5/beta-hcl2-note.mdx'`
+
+The special `packer` configuration block type is used to configure some
```

Comment:
Special makes it seem like a temporal or workaround block to me. What do you think about dropping "special". 
```suggestion
The `packer` configuration block type is used to configure some
```

Reply:
Yup I agree !